dadjokes-onchain/foundry/test/DadJokes.t.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+import "../src/DadJokes.sol";
+
+contract DadJokesTest is Test {
+    DadJokes private dadJokes;
+    address private creator1 = address(1);
+    address private creator2 = address(2);
```

Comment:
since creator2 does not appear to be authoring jokes in the test, consider renaming to user1 or something?